extensions/amp-social-share/0.1/amp-social-share.css

```diff
@@ -19,51 +19,44 @@
 amp-social-share {
```

Comment:
Dropped width height from here. Should I drop the whole selector and move other styles to amp.css?